recipe/bld.bat

```diff
@@ -0,0 +1,6 @@
+cargo build --release
+cargo build --release --manifest-path crates\c-api\Cargo.toml
+
+copy %SRC_DIR%\target\release\wasmtime.exe %LIBRARY_BIN%
+copy %SRC_DIR%\target\release\wasmtime.dll  %LIBRARY_BIN%
```

Comment:
Does it have a companion `.lib` file?

Reply:
@JeanChristopheMorinPerso it has following files:

17.08.2023  16:48            51 315 wasmtime.d
17.08.2023  16:48        10 335 232 wasmtime.dll
17.08.2023  16:47            61 373 wasmtime.dll.exp
17.08.2023  16:47           101 722 wasmtime.dll.lib
17.08.2023  16:44        21 105 152 wasmtime.exe
17.08.2023  16:47        74 406 860 wasmtime.lib
17.08.2023  16:48         5 525 504 wasmtime.pdb

so the wasmtime.lib is static and wasmtime.dll.lib is import I believe, shall we take import only into %LIBRARY_LIB%  ?

Reply:
I frankly don't know. I'd look at how wasmtime-py links wasmtime. If everything links correctly, then we could just add a comment in the build script to say that if someone needs these files, then we could just push a PR to rebuild the package.

Reply:
wasmtime-py just uses dll only, see 
https://github.com/AnacondaRecipes/wasmtime-py-feedstock/pull/1/files#diff-6575ceeaf967f9e78afe3221eb5d5034d1edc55ffd549e28e7a1a62c02bd2820

Reply:
@JeanChristopheMorinPerso  according to https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-creation the dll.lib is used in build process, so if building apps by customers is a valid use case ( which I do not know) we should include it also

Reply:
>wasmtime-py just uses dll only, see
https://github.com/AnacondaRecipes/wasmtime-py-feedstock/pull/1/files#diff-6575ceeaf967f9e78afe3221eb5d5034d1edc55ffd549e28e7a1a62c02bd2820

I would look at the builds logs if it was me.

>@JeanChristopheMorinPerso according to https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-creation the dll.lib is used in build process, so if building apps by customers is a valid use case ( which I do not know) we should include it also

Yes, and that's why I said "if someone needs these files, then we could just push a PR to rebuild the package"